### ISSUE TITLE
SQL Explorer

### ISSUE THREAD

**Issue by meetjaydeep** [roslynpad/roslynpad#94]
Add SQL Explorer feature so that we can directly run LINQ query on DB.
Similar feature in Linqpad

**Comment by aelij** [comment]
Thanks for the suggestion!

However, I must say this is a low priority for me, so unless someone contributes the code, it won't happen any time soon :)

**Comment by SlowLogicBoy** [comment]
I would suggest plugin api and make plugins like these, because someone wants SQL Explorer, I want NoSQL explorer

**Comment by aelij** [comment]
Yep, that's already been suggested, but a plugin API also requires a lot of effort (designing a stable API, a repository, etc.), and that's another low priority for me right now. Maybe some day :)

**Comment by arosas** [comment]
I would be more incline to use RoslynPad if it had some way to connect to SQL Server DB.  Is there a plugin at least for it?

**Comment by aelij** [comment]
I'm just one developer doing this on my free time, so I have to prioritize, and unfortunately this is a feature not many have asked for, nor one I personally need.

I've not tried it, but you may be able to use EF Core's scaffolding tools to generate a DbContext and just `#load` it in RoslynPad.
